lib/mobilecommons.js

```diff
@@ -59,3 +62,28 @@ exports.profile_update = function (profileId, phone, oip, updateFields) {
     }
   });
 };
+
+/**
+ * Send a message to the given phone number with the given message.
+ *
+ * @param  {string} phoneNumber Phone number to send msg to.
+ * @param  {string} message     Message to send.
+ */
+exports.send_message = function (phoneNumber, message) {
+  const data = {
+    auth,
+    form: {
+      phone_number: phoneNumber,
+      body: message,
+      campaign_id: masterCampaignid,
+    },
+  };
+
+  const requestRetry = new RequestRetry();
+  requestRetry.setRetryConditions([400, 408, 500]);
+  requestRetry.post(`${uri}/send_message`, data, (error) => {
```

Comment:
Hi @sergii-tkachenko @jamjensen @classicfreddie @deadlybutter @justkika  -- just wanted to make sure it's intentional that we're sending Broadcast Declined and Relative Reminder messages as SMS, not MMS. Is the desired behavior?  If they have copy set to over 255 characters, they will be truncated...